llama_index/agent/openai_assistant_agent.py

```diff
@@ -181,6 +184,8 @@ def from_new(
         # process files
         files = files or []
         file_dict = _process_files(client, files)
+        all_file_ids = list(file_dict.keys())
+        files_ids.extend(files_ids or [])
```

Comment:
do we ned this line? seems we just need all_file_ids

Reply:
Fixed :)